samples/openthread/coprocessor/README.rst

```diff
@@ -117,15 +118,22 @@ Building and running
 
 .. _ot_coprocessor_testing:
 
+HCI support
+===========
+
+Currently HCI is supported only via nRF native USB interface. The device will show two virtual UART ports.
+Usually the first one will be associated with HCI interface, the second one with Thread coprocessor.
+
 Testing
 =======
 
 After building the sample and programming it to your development kit, complete the following steps to test it:
 
-1. Connect the development kit's SEGGER J-Link USB port to the PC USB port with a USB cable.
+1. Connect the development kit's SEGGER J-Link USB port or native USB port to the PC USB port with a USB cable.
 #. Get the kit's serial port name (for example, :file:`/dev/ttyACM0`).
 #. Run and configure ot-cli as described in :ref:`ug_thread_tools_ot_apps`.
 #. From this point, you can follow the :ref:`ot_cli_sample_testing` instructions in the CLI sample by removing the `ot` prefix for each command.
+#. If you are using HCI please follow :ref`zephyrt:samples:bluetooth:hci_uart:README`:
```

Comment:
Is this meant to be step that happens after following the instructions for the CLI sample?

```suggestion
#. If you are using HCI, follow the instructions for the :ref`zephyrt:bluetooth-hci-uart-sample` sample in the Zephyr documentation.
```
Which section of the sample documentation is the user meant to follow?